utils/config/daos_agent.yml

```diff
@@ -65,7 +65,15 @@
 ## Note that this implies that stopping or restarting the agent will result
 ## in interruption of DAOS I/O for any local DAOS client processes that have
 ## an open pool handle.
-# disable_auto_evict: true
+## default: false
+#disable_auto_evict: true
+
+## Enable automatic eviction of open pool handles on agent startup. If enabled (disabled by
+## default), the agent will evict all open pool handles associated with the machine/host name
+## on agent start. This aims to garbage collect any handles not evicted after an agent or host
+## crash.
```

Comment:
Minor wording suggestion to remove some redundancy and irrelevant (to the user) details:
```suggestion
## If enabled, the agent will evict any open pool handles associated with this machine
## on agent startup. This allows the servers to reclaim resources that may not have been
## properly cleaned up in the event of an agent or machine crash.
```

Reply:
done